### ISSUE TITLE
Use Adapt for Intent Parsing

### ISSUE THREAD

**Issue by paddatrapper** [Progether/JAdventure#237]
Recently [Adapt](https://adapt.mycroft.ai/) by the Mycroft project has been open-sourced and I think it may be a good idea to use it to parse naturally written text into actionable intents. This would allow us to call actions based on a json script translation of the user's input as well as allow for the user to interact more naturally with the game.

The only drawback is that Adapt is written in Python, so we would need to use something like [Jython](https://wiki.python.org/jython/) to implement it. Thus creating two more dependencies - Jython and Adapt.

Adapt source code is available on [GitHub}(https://github.com/MycroftAI/adapt)

**Comment by Dev-Osmium** [comment]
I don't know if this is strictly necessary, but if people want it then we can try and implement it, but only much later.

**Comment by paddatrapper** [comment]
Yeah, it is purely an interesting nice-to-have, but nothing critical
